lightning/src/ln/channelmanager.rs

```diff
@@ -4908,7 +4965,7 @@ impl<Signer: Sign, M: Deref, T: Deref, K: Deref, F: Deref, L: Deref> ChannelMana
 					break Ok((raa_updates.accepted_htlcs, raa_updates.failed_htlcs,
 							raa_updates.finalized_claimed_htlcs,
 							chan.get().get_short_channel_id()
-								.expect("RAA should only work on a short-id-available channel"),
+								.unwrap_or(chan.get().outbound_scid_alias()),
```

Comment:
It looks like this could cause us to set `short_channel_id` in `PaymentPathFailed` to the alias (maybe elsewhere too), may want to document